python2/raygun4py/middleware/wsgi.py

```diff
@@ -33,33 +33,7 @@ def __call__(self, environ, start_response):
                 try:
                     iterable.close()
                 except Exception as e:
-                    request = self.build_request(environ)
+                    rg_request_details = http_utilities.build_wsgi_compliant_request(environ)
```

Comment:
Hey @ericb-granular, just porting this across to the Python 3 version atm so looking in a bit more detail. This variable doesn't seem to be used now? Did you miss updating the next line to pass it as the request argument? Just unsure if I'm missing something.

Reply:
Oh snap - yep, good eye. Line 37 needs to be: 
```
self.sender.send_exception(exception=e, request=rg_request_details)
```

Reply:
Great, thought that was the case. Just wanted to check I wasn't missing some Python voodoo haha.